nextmv/nextmv/cloud/application.py

```diff
@@ -1766,6 +1771,11 @@ def __update_app_binary(
                 )
             )
 
+    def __console_url(self, run_id: str) -> str:
+        """Auxiliary method to get the console URL for a run."""
+
+        return f"https://cloud.nextmv.io/app/{self.id}/run/{run_id}?view=details"
```

Comment:
Doesn't this (kind of) demand support for other endpoints/environments as well? Particularly, when considering current roadmap and priorities (+testing).

Maybe the `nextmv.cloud.Client` could have a custom `console_url` (in addition to the API `url`) that we use here?

Reply:
This is why you are my always trusted reviewer, because you catch things that I overlook for doing things too fast and loose 😄 .

Reply:
Naaww. :heart: